ansible/csm_packages.yml

```diff
@@ -77,3 +77,34 @@
           # More Cray services
           enable bos-reporter.service
           enable cfs-state-reporter.service
+
+# Application-nodes only play
+- hosts: Application:!cfs_image
+  gather_facts: yes
+  # Gather the minimum that gets the ansible_distribution variable set
+  gather_subset:
+    - '!all'
+    - '!min'
+    - distribution
+  any_errors_fatal: true
+  remote_user: root
+  vars_files:
+    - vars/csm_repos.yml
+    - vars/csm_packages.yml
+  roles:
+    # Install CSM repositories and packages
+    - role: csm.packages
+      vars:
+        packages: "{{application_csm_sles_packages }}"
+      when: ansible_distribution_file_variety == "SUSE"
+  tasks:
+    - name: Enable smart service
+      systemd:
+        name: smart
+        state: started
+        enabled: true
+    - name: Enable cray-node-exporter service
+      systemd:
+        name: cray-node-exporter
+        state: started
+        enabled: true
```

Comment:
```suggestion
```

Reply:
The rpms will get installed during image customization here:
https://github.com/Cray-HPE/csm-config/pull/192/files#diff-41d62977d5721c4697cdbbc7f13d22744c6522ccfd33887dc972339e45cd21adR68-R71

I made a proposal to enable the services during image customization as well similar to the other CFS services and then removed the duplicate rpm install during node personalization.

This should speed up the CSM CFS as it won't need to run during node personalization. NERSC seems to track how much time is spent during node personalization so if we can avoid zypper commands during boot, that will help.